src/test/java/alinaTinca/tests/Tema5/Demoqa/Elements/BrokenLinksImagesClass.java

```diff
@@ -0,0 +1,38 @@
+package alinaTinca.tests.Tema5.Demoqa.Elements;
+
+import AlinaTinca.BrowserManagerClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BrokenLinksImagesClass {
+
+    public static void main(String[] args) {
+        goToTheBrokenLinksImages();
+        //check if the method called goToTheBrokenLinksImages is done successfully
+    }
+
+    public static void goToTheBrokenLinksImages(){
+        //create a method called goToTheBrokenLinksImages that goes to the Broken Links-Images section and identifies elements and make operations on them
+        ChromeDriver driver = BrowserManagerClass.createChromeDriver();
+        //create and configure the Chrome browse page settings
+        driver.get("https://demoqa.com/broken");
+        //get to the Demoqa URL
+        WebElement BrokenLinksImagesSection = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[7]/span"));
+        //find the section called Broken Links Images
+        BrokenLinksImagesSection.click();
+        //click on this section
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        js.executeScript("window.scrollBy(0,document.body.scrollHeight)", "");
```

Comment:
Scrolling it's not working. Please double check it. Maybe consider using Actions class. This is the exception ->
![image](https://user-images.githubusercontent.com/18215854/211497810-010b8d9e-76f1-436c-9888-49acece322a3.png)